docs/usage/14-testing.md

```diff
@@ -116,7 +111,112 @@ def test_health_check():
         assert response.text == "healthy"
 ```
 
-## Create a Test Request
+## Using pydantic-factories
+
+Starlite bundles the library [pydantic-factories](https://github.com/Goldziher/pydantic-factories), which offers an easy and powerful way to generate mock data from pydantic models and dataclasses.
+
+Let's say we have an API that talks to external service (that we don't care about in unit tests) and retrieves some item:
+
+```python title="main.py"
+from typing import Protocol, runtime_checkable
+
+import pytest
+from pydantic import BaseModel
+from starlette.status import HTTP_200_OK
+from starlite import Provide, create_test_client, get
+
+
+class Item(BaseModel):
+    name: str
+
+
+@runtime_checkable
+class Service(Protocol):
+    def get(self) -> Item:
+        ...
+
+
+@get(path="/item")
+def get_item(service: Service) -> Item:
+    return service.get()
+```
+
+Then we can add a test for `/item` route:
+
+```python title="main.py"
+@pytest.fixture
+def item():
+    return Item(name="Chair")
+
+
+def test_get_item(item: Item):
+    class MyService(Service):
+        def get_one(self) -> Item:
+            return item
+
+    with create_test_client(
+        route_handlers=get_item,
+        dependencies={"service": Provide(lambda: MyService()),
+    }) as client:
+        response = client.get("/item")
+        assert response.status_code == HTTP_200_OK
+        assert response.json() == item.dict()
+```
+
+As you can see, we came up with a test product by ourselves. It is fine for small applications, but can be quite a pain in larger ones. That's where [pydantic-factories](https://github.com/Goldziher/pydantic-factories) library comes in. It generates mock data for pydantic datastructures based on type annotations. Let's rewrite our example using it:
```

Comment:
```suggestion
While we can define the test data manually, as is done in the above, this can be quite cumbersome. That's where [pydantic-factories](https://github.com/Goldziher/pydantic-factories) library comes in. It generates mock data for pydantic models and dataclasses based on type annotations. With it, we could rewrite the above example like so:
```